backend/app/views/spree/admin/payments/_form.html.erb

```diff
@@ -1,19 +1,19 @@
 <div data-hook="admin_payment_form_fields" class="row">
   <div class="alpha three columns">
     <div class="field">
-      <%= f.label :amount, Spree.t(:amount) %>
+      <%= f.label :amount %>
       <%= f.text_field :amount, :value => @order.display_outstanding_balance.money, :class => 'fullwidth' %>
     </div>
   </div>
   <div class="omega nine columns">
     <div class="field">
-      <label><%= Spree.t(:payment_method) %></label>
+      <label><%= Spree::PaymentMethod.model_name.human %></label>
       <ul>
         <% @payment_methods.each do |method| %>
-          <li data-id="<%= Spree.t(method.name, :scope => :payment_methods, :default => method.name).parameterize %>">
+          <li data-id="<%= Spree.t("#{method.name.downcase.tr(" ", "_")}").parameterize %>">
             <label data-hook="payment_method_field">
               <%= radio_button_tag 'payment[payment_method_id]', method.id, method == @payment_method, { class: "payment_methods_radios" } %>
-              <%= Spree.t(method.name, :scope => :payment_methods, :default => method.name) %>
+              <%= Spree.t("#{method.name.downcase.tr(" ", "_")}") %>
```

Comment:
There is no payment_methods scope in the base solidus dictionary and the default PaymentMethods are named `"Credit Card"`, `"Store Credit"`, and `"Check".  Only Check is close to the syntax we are using in the I18n dictionary for keys.  The current way it works just by abusing the functionality of default not throwing a translation missing when it's used.  This manipulation was the way I thought of actually using keys in the dictionary though I agree it isn't the cleanest approach.